### ISSUE TITLE
Most Chinese games can't be played 

### ISSUE THREAD

**Issue by yoyoliyang** [Gericom/GBARunner2#27]
As the title, A lot of Sinicization games can't be played.

**Comment by FlameKat53** [comment]
Most of my iQue games work, Famicom Collection is broken though, and Polarium Advance CHN (And all other regions, too) got broken

**Comment by Summerslyb** [comment]
If the game contains a custom title, it must be removed.

For example, I try to remove the custom title for "The Legend of Zelda - A Link to the Past and Four Swords". It works normally.
However, it requires using no$gba for debugging dynamically_(:3」∠)_

What's more, many pokemon mod cannot work even if the title was removed.
I don't know why a game works well on the simulator, but not work for real console.

**Comment by Summerslyb** [comment]
![QQ截图20200112223817](https://user-images.githubusercontent.com/10459559/72221217-75bce600-3593-11ea-8992-f21f78103ac4.png)
A example of custom title code. @Gericom

**Comment by zenseii** [comment]
You probably shouldn't share that rom here. Otherwise good job!

**Comment by Gericom** [comment]
Wait, it seems to use the STRT instruction (which is no different from STR on gba). I do not handle that instruction in the abort handler currently.

**Comment by zenseii** [comment]
Btw @Summerslyb the ROM is still available in the history of your post. I'm not sure how to remove that actually...

**Comment by Gericom** [comment]
That "title code" this was referring to rom intros which appearently use strt/ldrt sometimes. It is unrelated to chinese games. If the chinese games just work when clean I can probably close this issue and make a separate one for ldrt/strt support

**Comment by Summerslyb** [comment]
Chinese game is a complicated issue. Different games may have different problems.

“Custom title” is just a common problem I found.
After removing the title, some games will still not work, which may require further testing.

Anyway, I think it is necessary to open a new Issue for ldrt/strt support.

**Comment by Summerslyb** [comment]
> Btw @Summerslyb the ROM is still available in the history of your post. I'm not sure how to remove that actually...

I deleted the history permanently.

**Comment by Summerslyb** [comment]
I try to modify the "strt/ldrt" to "str/ldr". It will no longer get stuck. 
But the screen doesn't show anything useful but stays blank. 
And the interrupt for touch on bottom screen is blocked util you enter the key. 

![QQ截图20200216145216](https://user-images.githubusercontent.com/10459559/74600397-07a49b00-50cc-11ea-8cb1-0c606ac04588.png)

**Comment by Gericom** [comment]
Did you try with center and mask disabled?

**Comment by Summerslyb** [comment]
Oh, when disabled 'center and mask', the title showed. It may related to the frame delay.

**Comment by Gericom** [comment]
No, the reason is that that title appearently doesn't use any irqs which is the issue. I will implement support for ldrt and strt and then close this issue

**Comment by qamichelemassa5859** [comment]
When getting all WORK on mGBA v.0.9.3 on my PC, the game still crashing in previous GBARunner2 versions, but has still doesn't work anymore in latest mGBA3DS versions.

**Comment by SLKun** [comment]
> When getting iQue famicom mini collection game TO WORK on mGBA v.0.9.3 on my PC, the game still crashing in previous GBARunner2 versions, but has still doesn't work anymore in latest mGBA3DS versions.

The famicom mini collection is not supported by GBARunner2.

**Comment by qamichelemassa5859** [comment]
> > When getting iQue famicom mini collection game TO WORK on mGBA v.0.9.3 on my PC, the game still crashing in previous GBARunner2 versions, but has still doesn't work anymore in latest mGBA3DS versions.
> 
> The famicom mini collection is not supported by GBARunner2.

Goodbye Gericom ;( for making any sense. Just use mGBA3DS instead
